internal/featuresupport/features.go

```diff
@@ -172,10 +172,6 @@ func (feature *ClusterManagedNetworkingFeature) GetName() string {
 }
 
 func (feature *ClusterManagedNetworkingFeature) GetSupportLevel(filters SupportLevelFilters) models.SupportLevel {
-	if swag.StringValue(filters.CPUArchitecture) == models.ClusterCPUArchitectureS390x || swag.StringValue(filters.CPUArchitecture) == models.ClusterCPUArchitecturePpc64le {
```

Comment:
why did you remove this code? it should be unavailable now

Reply:
fixed